package/rmkit/package

```diff
@@ -2,7 +2,7 @@
 # Copyright (c) 2020 The Toltec Contributors
 # SPDX-License-Identifier: MIT
 
-pkgnames=(harmony mines nao remux simple)
+pkgnames=(harmony minesweeper mines nao remux simple)
```

Comment:
it's a little confusing that `minesweeper` (the deprecated name) is placed at toplevel and `mines` is added as `depends` for minesweeper

Reply:
So if you have minesweeper installed it will automatically install `mines` which will then remove `minesweeper`. Eventually we can drop the `minesweeper` package.